source/features/copy-file.tsx

```diff
@@ -22,18 +23,28 @@ function handleClick({delegateTarget: button}: delegate.Event<MouseEvent, HTMLBu
 }
 
 function renderButton(): void {
-	for (const button of select.all('.file-actions .btn, [data-hotkey="b"]')) {
-		button
-			.parentElement! // `BtnGroup`
-			.prepend(
-				<button
-					className="btn btn-sm tooltipped tooltipped-n BtnGroup-item rgh-copy-file"
-					aria-label="Copy file to clipboard"
-					type="button"
-				>
-					Copy
-				</button>
-			);
+	for (const button of select.all<HTMLAnchorElement>([
+		'.file-actions .btn[href*="/raw/"]', // `isGist`
+		'[data-hotkey="b"]'
+	])) {
+		const copyButton = (
+			<button
+				className="btn btn-sm tooltipped tooltipped-n BtnGroup-item rgh-copy-file"
+				aria-label="Copy file to clipboard"
+				type="button"
+			>
+				Copy
+			</button>
+		);
+		if (pageDetect.isGist() && button.href.endsWith('.svg')) { // SVG files have a render blob button
```

Comment:
Without this

![image](https://user-images.githubusercontent.com/16872793/86920496-e20a4380-c0f7-11ea-949d-767968ae8e29.png)

Reply:
> SVG files have a render blob button

It's not exclusive to SVGs; use a DOM-based detection

Reply:
Do you have another example that I could test on? I tried a few files and could not find another one.

Reply:
Markdown is one, through our `view-markdown-source` feature

Reply:
Possibly these too https://docs.github.com/en/github/managing-files-in-a-repository/working-with-non-code-files

Reply:
> Markdown is one, through our `view-markdown-source` feature

`view-markdown-source` is not enabled (and cannot be) on gists.

I did a dom detection, let me know if you want me to improve anything.